### DISCUSSION TITLE
726

### DISCUSSION THREAD

**Discussion by giscus** [jojoldu/blog-comments#909] [Comments]
# 726

2023년 6월 26일, JetBrains에서 AI Assistant 기능이 추가 되었다. JetBrains IDE의 AI Assistant 현재 (2023.07.12) 까지 정식 기능은 아니며 EAP 버전에서만 사용 가능하다. GPT 채팅이 있어서 IDE에서는 어떤 차별점이 있을까 궁금해서 EAP 버전을 설치해서 사용해보기 시작했다. 아래 모든 단축키는 Mac을 기준으로 한다. 1. 설치 IntelliJ나 WebStorm 등 IDEA 에서는 다 지원하기 때문에 어떤 IDE라도 EAP 버전을 설치하면 바로 사용할 수 있다. EAP 버전을 설치하는 방법은 JetBrains Toolbox 를 설치하고, 이를 실행해보면 다음과 같이 설치된 IDE들의 목록이 나온다. 여기서 EAP 버전을 설치하고 싶은 IDE의..

https://jojoldu.tistory.com/726

**Comment by chucoding** [comment]
AI 어시스턴트는 코드 분석을 위해 어떤식으로 데이터 수집을 하는지, 회사에서 쓸 수 있는지 궁금하네요.

**Reply by highRPM** [reply]
JetBrains는 철저한 실사를 수행하며 사용자의 데이터를 수집하거나 이를 이용해 학습하는 LLM 제공자와는 협력하지 않습니다. 현재, AI Assistant는 자체 모델과 OpenAI를 기반으로 합니다. 라고 하네요

**Comment by jws1837** [comment]
메일로 ai assisist기능 안내 소식이 와서 관련글 검색하다가 들어왔습니다. 역시 빠르게 체험해보셨군요. 
감사합니다.
